### DISCUSSION TITLE
Cannot import mssql driver

### DISCUSSION THREAD

**Discussion by staff0rd** [mikro-orm/mikro-orm#5462] [General]
### Describe the bug

I can import postgres/mysql drivers, but not the mssql driver:

```typescript
import { PostgreSqlDriver } from '@mikro-orm/postgresql' //  ✅ no problem
import { MySqlDriver } from '@mikro-orm/mysql' // ✅ no problem
import { MsSqlDriver } from '@mikro-orm/mssql' // ❌ error!
```

The error is:

```bash
src/mikro-orm.config.ts:2:38 - error TS7016: Could not find a declaration file for module '@mikro-orm/mssql'. '/home/stafford/git/spacetraders-again/node_modules/@mikro-orm/mssql/index.mjs' implicitly has an 'any' type.
  There are types at '/home/stafford/git/spacetraders-again/node_modules/@mikro-orm/mssql/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@mikro-orm/mssql' library may need to update its package.json or typings.

2 import { MsSqlDriver, Options } from '@mikro-orm/mssql'
```

I would expect the mssql driver package to import without error, similar to the postgres/mysql driver packages.

### Reproduction

via the `mssql-test` branch of [this repo](https://github.com/staff0rd/spacetraders-again/tree/mssql-test)

```bash
git clone https://github.com/staff0rd/spacetraders-again
cd spacetraders-again
git checkout mssql-test
npm i
npm run check-types
```

### What driver are you using?

@mikro-orm/mssql

### MikroORM version

6.2.1

### Node.js version

v20.10.0

### Operating system

Ubuntu 22.04.3 (WSL)

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
This should help: 02494bf129e15261ba17218288a840120fedb089
